eth2near/eth2near-block-relay-rs/olga_local_test_config.toml

```diff
@@ -0,0 +1,10 @@
+beacon_endpoint = "https://lodestar-kiln.chainsafe.io"
+eth1_endpoint = "https://rpc.kiln.themerge.dev"
+total_submit_headers = 4
+near_endpoint = "https://rpc.testnet.near.org"
+signer_account_id = "olga24912.testnet"
+path_to_signer_secret_key = "/home/olga/.near-credentials/testnet/olga24912.testnet.json"
+contract_account_id = "dev-1659151769901-77594397083671"
+network = "kiln"
+contract_type = "near"
+light_client_updates_submission_frequency_in_epochs = 2
```

Comment:
This file is redundant. If it's needed, it should be called `local_test_config.toml` and then should be in `.gitignore`.

Reply:
Fixed in 40e8b0f439def64dff0c44ac0c18f6f3172f75f7 and 1ed11716c2ebb1227c7d8056398d97520ab5bbc7